### PR TITLE
feature/SC-7025 - create new component tapbar

### PR DESCRIPTION
**Pull request by xavero-evandro**
# Description

I as a User want to have a tab bar to switch between tabs.

Acceptance Criteria:

if possible use the existing component from the storybook
if not, create a new component
empty tab bar

## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/SC-7025

## Changes

Added new layout to tab bar

## Screenshots of UI changes
image
image

## Approval for review

- [X] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by xavero-evandro** [comment]
@schul-cloud-bot update screenshots

File: src/components/atoms/Tab.vue [hunk 1]
```diff
@@ -21,6 +21,10 @@ export default {
 		selected: {
 			type: Boolean,
 		},
+		iconName: {
```
Comment (pkleybolte) [inline src/components/atoms/Tab.vue:24]: I suggest a unit test to check that the icon is displayed if this property is set

File: src/components/organisms/Tabs/Tabs.vue [hunk 2]
```diff
@@ -50,44 +53,83 @@ export default {
...
+			.tab-button,
+			i {
+				font-weight: var(--font-weight-bold);
+				color: var(--color-gray-dark);
+				transition: var(--duration-transition-slow) ease-in-out;
 			}
 			&::after {
 				position: absolute;
-				bottom: -2px;
+				bottom: -0.2rem;
```
Comment (pkleybolte) [inline src/components/organisms/Tabs/Tabs.vue:96]: This is ok, but I'd prefer `var(--space-xs-3)`

File: src/components/atoms/Tab.vue (hunk 1 above)
Comment (xavero-evandro) [reply src/components/atoms/Tab.vue:24]: Created new unit test!

File: src/components/organisms/Tabs/Tabs.vue (hunk 2 above)
Comment (xavero-evandro) [reply src/components/organisms/Tabs/Tabs.vue]: Changed to var(--space-xs-3)

**Review by pkleybolte — DISMISSED** [review]

**Comment by xavero-evandro** [comment]
@schul-cloud-bot update screenshots

**Review by pkleybolte — APPROVED** [review]

**Pull request merged**